### PR TITLE
Fixes #201 Adds functionality to Sensor QuickView

### LINKED ISSUE #201: Adding functionalities to Sensor QuickView
**Actual Behaviour**

As of now, there is just a UI for the Sensor QuickView. Functionalities to buttons and science methods haven't been used so far.

**Expected Behaviour**

The feature should enable the user to detect a sensor and get the raw data from that sensor.

**Steps to reproduce it**

Main menu -> Applications -> Sensor QuickView

**LogCat for the issue**

None

**Screenshots of the issue**

The textviews look odd as of now, but will be proper when filled with actual data.

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by asitava1998**
Fixes issue #201

Changes: 
* Added scienceLab methods and functions to buttons, spinners etc.
* As of now, no I2C sensors are available with me for testing. As soon as I receive my order of sensors, the code will be tested and further changes will be made.
* Some minor change in a few sensor files.
Screenshots for the change:

### CODE REVIEW

File: app/src/main/java/org/fossasia/pslab/activity/SensorActivity.java [hunk 1]
```diff
@@ -3,25 +3,321 @@
...
+        buttonSensorGetRaw.setEnabled(false);
+
+        buttonSensorAutoscan.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                ArrayList<Integer> data = new ArrayList<Integer>();
+                ArrayList<String> datastr = new ArrayList<String>();
+                String datadisp = "";
+                try {
+                    i2c.init();
```
Comment (jithinbp) [inline app/src/main/java/org/fossasia/pslab/activity/SensorActivity.java]: init does not need to be called here

Comment (asitava1998) [inline app/src/main/java/org/fossasia/pslab/activity/SensorActivity.java]: Just the scan method should only be called, right?

Comment (jithinbp) [inline app/src/main/java/org/fossasia/pslab/activity/SensorActivity.java]: try logging the results of I2C scan function

Comment (jithinbp) [inline app/src/main/java/org/fossasia/pslab/activity/SensorActivity.java]: Yes. but init should not make a difference.
I think the problem with reading single bytes is probably causing issues here. Check the logs

Comment (asitava1998) [inline app/src/main/java/org/fossasia/pslab/activity/SensorActivity.java]: I don't have any I2C sensors as of now to actually try out the code but I did order `MPU6050` and `HMC5883L` . I will test this soon.

**Comment by jithinbp** [comment]
You don't a sensor to check the autoscan method though.
The onboard DAC is an I2C device with address 96 . It should be detected even if nothing is plugged in externally

**Comment by asitava1998** [comment]
@jithinbp Then let me check that autoScan method then.

**Comment by viveksb007** [comment]
@asitava1998 getAcknowledgement method is used during scanning. Are you getting any read error in the logs or its working fine?

**Comment by asitava1998** [comment]
No errors as of now

**Pull request merged**